sop-website/docs/Waveform-Data/Waveform-Extension.mdx

```diff
@@ -0,0 +1,222 @@
+| 10    | waveform_feature_end_timestamp   | time      | Recommended            | End time of the window. Required for interval-based features like HRV, respiratory rate, or entropy. Equal to start time for instantaneous features.                                                                                                                                                                                                                           |
+| 11    | is_feature_overflow              | boolean   | Optional               | Populate as TRUE if the feature was derived from signal segments that span multiple waveform files, waveform occurrences, or signal channels. This field helps downstream consumers account for composite or stitched features that cannot be strictly attributed to a single source. Leave NULL if undetermined.                                                              |
+| 12    | value_as_number                  | float     | Recommended            | Populate if the feature is quantitative (e.g., HR = 75 bpm, Entropy = 0.85). Must be a valid float.                                                                                                                                                                                                                                                                            |
+| 13    | value_as_concept_id              | int       | Recommended            | Use if the feature is categorical (e.g., “Low signal quality”, “Apnea present”). Map to OMOP concept or use custom 2B ID.                                                                                                                                                                                                                                                      |
+| 14    | value_as_string                  | string    | Optional               | Use if the feature cannot be mapped or is stored in descriptive form (e.g., “artifact detected”, “tachycardia”). Supports flexibility in early-stage pipelines.                                                                                                                                                                                                                |
+| 15    | value_is_a_registry_file         | boolean   | No                     | Set to TRUE (1) if the feature value is stored as a file in WAVEFORM_REGISTRY, rather than a discrete number or concept (e.g., time-frequency embedding, long-form entropy sequence). When true, link to the file using waveform_feature_id. Leave as NULL or FALSE if the value is numeric, categorical, or string-based. Helps differentiate scalar vs. file-based features. |
```

Comment:
Shouldn't this always be FALSE if the feature isn't stored as a file under the Waveform Registry table? Is there a scenario where NULL would be more appropriate?